product_docs/docs/pgd/5.6/deploy-config/deploy-cloudservice/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Deploying and configuring PGD on EDB Postgres AI Cloud Service
+navTitle: On EDB Cloud Service
+redirects:
+  - /pgd/latest/deploy-config/deploy-biganimal/ #generated for pgd deploy-config-planning reorg
+  - /pgd/latest/install-admin/admin-biganimal/ #generated for pgd deploy-config-planning reorg 
+  - /pgd/latest/admin-biganimal/ #generated for pgd deploy-config-planning reorg 
+---
+
+EDB Postgres AI Cloud Service is a fully managed database-as-a-service with built-in Oracle compatibility. It runs in your cloud account where it's operated by our Postgres experts. EDB Postgres AI Cloud Service makes it easy to set up, manage, and scale your databases. The addition of distributed high-availability support powered by EDB Postgres Distributed (PGD) enables single- and multi-region Always-on clusters.
+
+This section covers how to work with EDB Postgres Distributed when deployed on EDB Postgres AI Cloud Service:
+
+- [Creating a distributed high-availability cluster](/edb-postgres-ai/cloud-service/getting_started/creating_cluster/creating_a_dha_cluster/) in the Cloud Service documentation works through the steps needed to:
+  - Prepare your cloud environment for a distributed high-availability cluster.
+  - Sign in to Cloud Service.
+    - Create a distributed high-availability cluster, including:
```

Comment:
The indentation feels a bit odd here. 
Maybe this bullet point should not be indented with four empty spaces, but only two. And the following two can keep the four-spaced indentation as they are part of "Create a distributed high-availability cluster"? 
```suggestion
  - Create a distributed high-availability cluster, including:
```